lib/component.ex

```diff
@@ -92,4 +92,47 @@ defmodule LiveSvelte do
   end
 
   defp id(name), do: "#{name}-#{System.unique_integer([:positive])}"
+
+  @doc false
+  def get_props(assigns) do
+    prop_keys =
+      assigns
+      |> Map.get(:__changed__)
+      |> Map.keys()
+
+    assigns
+    |> Enum.filter(fn
+      {:svelte_opts, _v} -> false
+      {k, _v} -> k in prop_keys
+    end)
+    |> Enum.into(%{})
```

Comment:
you can filter a map:
There is https://hexdocs.pm/elixir/1.14/Map.html#filter/2

Reply:
Thank you! Fixed here: https://github.com/woutdp/live_svelte/pull/38/commits/7c912171d4792f6c6f1197c1e4883144000666eb